### ISSUE TITLE
Google Maps v3 fix

### ISSUE THREAD

**Issue by tcmb** [geosdi/GWT-OpenLayers#39]
TL;DR: Updated version of accepted answer in https://stackoverflow.com/questions/32698848/openlayers-2-13-1-with-google-maps

I wrestled a bit with getting Google Maps v3 Layers to work. The problem I had was that instead of Google Maps tiles I just got a white/grey canvas.
I found that in the Google Maps showcase the version of the JS lib downloaded from Google was actually `v=3.21` in the source code of the showcase site, although the text on the showcase itself just mentions `v=3`.

I suppose this was done because of a change in the Google Maps JS API which broke OpenLayers 2. Pinning Maps JS to 3.21 still kept it working. But this fix no longer works since 3.21 is now retired.

To fix the fix, so to speak, there is a 'monkeypatched' version of Openlayers 2.13.1 which can be downloaded from https://drive.google.com/file/d/0B00jfOQmQSegYXRrSTJVTXZnNW8/view

I found this answer here: https://stackoverflow.com/questions/32698848/openlayers-2-13-1-with-google-maps
The required change itself, should the file linked above stop working, can also be found in this thread in the OpenLayers issue tracker: https://github.com/openlayers/openlayers/pull/1450#issuecomment-137508529
